src/ui/public/agg_types/__tests__/param_types/_calculate_interval.js

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 import expect from 'expect.js';
 import ngMock from 'ng_mock';
-import VisProvider from 'ui/vis';
+import { VisProvider } from 'ui/vis';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
 import AggTypesParamTypesCalculateIntervalProvider from 'ui/agg_types/param_types/_calculate_interval';
```

Comment:
AFAICT, there is no file corresponding to `ui/agg_types/param_types/_calculate_interval` in the source code. I looked in `https://github.com/elastic/kibana/tree/master/src/ui/public/agg_types/param_types`. How is this even working? 😄